ddtrace/contrib/dogpile_cache/region.py

```diff
@@ -38,4 +44,8 @@ def _wrap_get_create_multi(func, instance, args, kwargs):
         span.set_tag("keys", keys)
         span.set_tag("region", instance.name)
         span.set_tag("backend", instance.actual_backend.__class__.__name__)
-        return func(*args, **kwargs)
+        response = func(*args, **kwargs)
+        if isinstance(response, Iterable):
```

Comment:
What are the possible return values for `func(...)`? Can we document a case where it's not an iterable

Reply:
Sequence[Any] for the wrapped function according to their API [here](https://dogpilecache.sqlalchemy.org/en/latest/api.html#dogpile.cache.region.CacheRegion.get_or_create_multi).

Reply:
removed this check